Common/Orders/OrderJsonConverter.cs

```diff
@@ -96,6 +96,8 @@ public static Order CreateOrderFromJObject(JObject jObject)
             order.Id = jObject["Id"].Value<int>();
             order.Status = (OrderStatus) jObject["Status"].Value<int>();
             order.Time = jObject["Time"].Value<DateTime>();
+            order.LastFillTime = jObject["LastFillTime"]?.Value<DateTime>();
+            order.LastUpdateTime = jObject["LastUpdateTime"]?.Value<DateTime>();
```

Comment:
Nice catch! We have a collection of tests for the `OrderJsonConverter`, would be nice to add an assert for each one of these